pkg/nexus/nexus_test.go

```diff
@@ -0,0 +1,377 @@
+package nexus
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAddArtifactValid(t *testing.T) {
```

Comment:
(minor) I applied the pattern to group my tests by the function they test.
I you want to apply that you could group  as follows:

```go
func TestAddArtifact(t *testing.T) {
	t.Run("Test valid...", func(t *testing.T) {
		....test here...
	})
	t.Run("Test invalid...", func(t *testing.T) {
		....test here...
	})
}

```

Reply:
Thanks, good point.